src/LightningExtension/AwaitTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Acquia\LightningExtension;
+
+use Drupal\DrupalExtension\Context\MinkContext;
+
+trait AwaitTrait {
```

Comment:
This is just gardening for the test framework. It's cleaner than importing MinkContext just to call iWaitForAjaxToFinish().